azure/durable_functions/tasks/timer_task.py

```diff
@@ -0,0 +1,47 @@
+from ..models.Task import Task
+
+
+class TimerTask(Task):
+    """Represents a pending timer.
+
+    All pending timers must be completed or canceled for an orchestration to complete.
+
+    Example: Cancel a timer
+    ```
+    timeout_task = context.df.create_timer(expiration_date)
+    if not timeout_task.is_completed():
+        timeout_task.cancel()
+    ```
+    """
+
+    def __init__(self, action, is_completed, timestamp, id_):
+        self._action = action,
```

Comment:
The "," make each of these instance variables into tuples, which may cause serialization problems

Reply:
@priyaananthasankar , I don't see this fixed in the latest PR